[2022-05-17 01:32:20,422] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-15T16:00:47.250021+00:00 [queued]>
[2022-05-17 01:32:20,434] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-15T16:00:47.250021+00:00 [queued]>
[2022-05-17 01:32:20,435] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-17 01:32:20,436] {taskinstance.py:1242} INFO - Starting attempt 5 of 8
[2022-05-17 01:32:20,437] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-17 01:32:20,450] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_run.model.jaffle_shop.int_orders> on 2022-05-15 16:00:47.250021+00:00
[2022-05-17 01:32:20,455] {standard_task_runner.py:52} INFO - Started process 260 to run task
[2022-05-17 01:32:20,459] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'model_level_dbt_dag', 'dbt_run.model.jaffle_shop.int_orders', 'scheduled__2022-05-15T16:00:47.250021+00:00', '--job-id', '541', '--raw', '--subdir', 'DAGS_FOLDER/model_level_dbt_dag.py', '--cfg-path', '/tmp/tmpqo0zxzjn', '--error-file', '/tmp/tmpi0lypjcr']
[2022-05-17 01:32:20,460] {standard_task_runner.py:77} INFO - Job 541: Subtask dbt_run.model.jaffle_shop.int_orders
[2022-05-17 01:32:20,512] {logging_mixin.py:109} INFO - Running <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-15T16:00:47.250021+00:00 [running]> on host 9d5de636d688
[2022-05-17 01:32:20,588] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=luciano.naveiro
AIRFLOW_CTX_DAG_ID=model_level_dbt_dag
AIRFLOW_CTX_TASK_ID=dbt_run.model.jaffle_shop.int_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-05-15T16:00:47.250021+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-15T16:00:47.250021+00:00
[2022-05-17 01:32:20,590] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-17 01:32:20,593] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt --no-write-json run --target dev --models int_orders --profiles-dir /opt/dbt/jaffle_shop --project-dir /opt/dbt/jaffle_shop']
[2022-05-17 01:32:20,608] {subprocess.py:85} INFO - Output:
[2022-05-17 01:32:24,278] {subprocess.py:89} INFO - 01:32:24  Running with dbt=1.1.0
[2022-05-17 01:32:24,428] {subprocess.py:89} INFO - 01:32:24  Found 5 models, 13 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-17 01:32:24,438] {subprocess.py:89} INFO - 01:32:24
[2022-05-17 01:32:24,656] {subprocess.py:89} INFO - 01:32:24  Concurrency: 1 threads (target='dev')
[2022-05-17 01:32:24,659] {subprocess.py:89} INFO - 01:32:24
[2022-05-17 01:32:24,677] {subprocess.py:89} INFO - 01:32:24  1 of 1 START view model dev.int_orders ......................................... [RUN]
[2022-05-17 01:32:24,879] {subprocess.py:89} INFO - 01:32:24  1 of 1 OK created view model dev.int_orders .................................... [[32mCREATE VIEW[0m in 0.20s]
[2022-05-17 01:32:25,004] {subprocess.py:89} INFO - 01:32:25
[2022-05-17 01:32:25,007] {subprocess.py:89} INFO - 01:32:25  Finished running 1 view model in 0.56s.
[2022-05-17 01:32:25,014] {subprocess.py:89} INFO - 01:32:25
[2022-05-17 01:32:25,017] {subprocess.py:89} INFO - 01:32:25  [32mCompleted successfully[0m
[2022-05-17 01:32:25,019] {subprocess.py:89} INFO - 01:32:25
[2022-05-17 01:32:25,021] {subprocess.py:89} INFO - 01:32:25  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-05-17 01:32:26,142] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-17 01:32:26,215] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=model_level_dbt_dag, task_id=dbt_run.model.jaffle_shop.int_orders, execution_date=20220515T160047, start_date=20220517T013220, end_date=20220517T013226
[2022-05-17 01:32:26,267] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-17 01:32:26,327] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
